.travis.yml

```diff
@@ -6,6 +6,7 @@ go:
 install:
   - make dependencies
   - test -z "$(go fmt ./...)"
+  - test -z "$(golint -min_confidence=1.0 ./modules/host)"
```

Comment:
I don't like this -- `golint` should be treated like `go vet`, meaning it should be run in the Makefile rather than Travis. Otherwise you won't know about a golint failure until you make a PR.